stl/inc/yvals_core.h

```diff
@@ -855,6 +855,42 @@
 
 // next warning number: STL4027
 
+// P0619R4 remove deprecated features from C++20
+#ifndef _HAS_REMOVED_FEATURES_CXX20
+#define _HAS_REMOVED_FEATURES_CXX20 (!_HAS_CXX20)
+#endif // _HAS_REMOVED_FEATURES_CXX20
+
+#ifndef _HAS_DEPRECATED_TYPETRAITS

```

Comment:
Naming: This should probably say `TYPE_TRAITS` with an underscore.

Reply:
Second that